.github/workflows/handle_llvm_runner_image.yml

```diff
@@ -9,53 +9,22 @@ on:
       - main
 
 jobs:
-  check_changes:
```

Comment:
As we have only one image, this is not required anymore in the first place. Secondly, it was not working as it should have, and anyway, 2 images were built even if only one changed. I'll return it back if required later.